plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java

```diff
@@ -926,6 +957,27 @@ public static String columnExtraInfo(boolean partitionKey)
         return partitionKey ? "partition key" : null;
     }
 
+    public static String erasePartitionColumnNames(String partitionName)
```

Comment:
This seems to duplicate `toPartitionValues`. Can we combine them?

Reply:
Changed to implement `erasePartitionColumnNames` in terms of `toPartitionValues`.